### ISSUE TITLE
Statistical analysis of calibration

### ISSUE THREAD

**Issue by sourishg** [sourishg/stereo-calibration#8]
1. Generate insightful plots describing camera calibration accuracy. Here's an example: plot the graph of reprojection error vs number of images used for calibration.

2. Compare multiple methods (AprilTags vs Checkerboard)

**Comment by BarunKGP** [comment]
How do I get the data? Do I run the code and store the data? Also, how do you want the comparison done?

**Comment by sourishg** [comment]
@BarunKGP yes you should collect the data by running the code. You can focus on solving (1) only for now since AprilTags calibration method for (2) isn't implemented yet.

I recommend using `matplotlib` for plotting.

**Comment by pushkalkatara** [comment]
The cam_left.yaml and the cam_stereo.yaml files are generated using opencv. Should i load these files using pyyaml in python and plot the graph in matplotlib or should i use matplotlib wrapper for c++?

**Comment by sourishg** [comment]
@pushkalkatara why are loading the yaml files? I think you only need to record the calibration error (which right now I'm echoing out in the terminal) and the number of images used for calibration (specified by the `-n` flag).

No need to write a C++ wrapper for `matplotlib`, you can use python.

**Comment by pushkalkatara** [comment]
@sourishg Please review the PR.

**Comment by BarunKGP** [comment]
@sourishg, should I embed python code in calib_intrinsic.cpp to use matplotlib and print out the graphs?
